README.md

```diff
@@ -43,7 +45,9 @@ OR
 
 Include **'therubyracer'** in your Gemfile.
 
-    gem "therubyracer", :require => 'v8'
+```bash
```

Comment:
@nickrivadeneira  These commands is default used by bash programs like CMD, terminal or command line.